packaging/rpm/ice.spec

```diff
@@ -203,13 +209,14 @@ network programming interfaces and allows you to focus your efforts on
 your application logic.
 
 #
-# ice-compilers package
+# ice2slice package
 #
-%package -n %{?nameprefix}ice-compilers
-Summary: Slice compilers for developing Ice applications
+%package -n %{?nameprefix}ice2slice
+Summary: The ice2slice compiler
 Requires: %{?nameprefix}ice-slice = %{version}-%{release}
-%description -n %{?nameprefix}ice-compilers
-This package contains Slice compilers for developing Ice applications.
+%description -n %{?nameprefix}ice2slice
+This package contains the ice2slice compiler to transforms Slice definitions in .ice files into
```

Comment:
This package contains the Ice-to-Slice compiler (ice2slice). The Ice-to-Slice compiler converts Slice definitions in .ice files into Slice definitions in .slice files. 
.ice is the Slice syntax and format understood by the Slice compilers provided by Ice;  .slice is the Slice syntax and format understood by slicec, the Slice compiler provided by IceRPC.